lib/include/openamp/virtio.h

```diff
@@ -111,6 +111,24 @@ struct virtio_buffer_info {
 	unsigned long size;
 };
 
+/**
+ * struct remoteproc_vring - remoteproc vring structure
```

Comment:
Seems like virtio_vring should go away, and just keep what we need in struct virtio_device directly.